src/eloqua.js

```diff
@@ -1,3 +1,7 @@
+/*
+* @version X.X.X
+*/
```

Comment:
Now that I'm looking at this, should we also give this file a proper header with the title of the plugin and the license information? We mentioned the MIT license in the package.json but I don't think we have any license text checked in. I think it would be standard to have a separate `LICENSE` file in the root of the repo, a quick comment in the readme, and then a link to this github repo (with mention of the license) at the top of this file.